tee-worker/litentry/core/assertion-build/src/dynamic/README.md

```diff
@@ -36,12 +36,48 @@ Should use the `dynamic` as your project root path in Remix IDE as below:
 remixd -s your_repo_path/tee-worker/litentry/core/assertion-build/src/dynamic --remix-ide https://remix.ethereum.org
 ```
 
-If you have not install remixd before, rub below script to install it.
+If you have not installed Remixd before, run the below script to install it.
 
 ```shell
 npm install -g @remix-project/remixd
 ```
 
+### Deploy
+
+The deployment script can be used to deploy a specific contract to the specified chain. Below is the usage of the deployment script.
+
+#### Command Syntax
+
+```shell
+pnpm run deploy-contract --contract <ContractName> --chain <ChainName> --secrets <Secret1> <Secret2> ...
+```
+
+#### Parameters
+
+-   --contract: Specify the name of the contract you wish to deploy.
+-   --chain: Specify the target chain environment. Supported values are:
+    -   local
+    -   dev
+    -   staging
+    -   prod
+-   --secrets: Provide the required secret values for the contract. These may include API keys, private keys, or other sensitive information needed for the deployment contract, multiple secrets must be separated by blank.
+
+#### Example
+
+To deploy the `TokenMapping` contract to the `dev` chain with specific secrets, you would run the following command:
+
+```shell
+pnpm run deploy-contract --contract TokenMapping --chain dev --secrets abc vna poi
```

Comment:
Can secret itself have space?

Reply:
Added support for passing secret with space